pymodbus/repl/client/main.py

```diff
@@ -37,9 +37,11 @@
     ModbusSocketFramer,
 )
 
+_logger = logging.getLogger()
+
 click.disable_unicode_literals_warning = True
 
-TITLE = f"""
+TITLE = r"""
```

Comment:
@janiversen  With raw string , the inline replacement of `{version}` does not work. we end up with literal `{version}` in the output instead of the actual output.  We need to revert back ti `f-strings` here

Reply:
Ok, but then I need to add some pylint disable.